index.html

```diff
@@ -8,9 +8,42 @@
     <link rel="stylesheet"  href="style.css">
 </head>
 <body>
-    <header>
-        
+    <div class ="porfolio_container">
+    
+    <header >
+        <nav >
+            <ul class ="top_navigation">
+                <li class ="nameplate"> 9:41</li>
+                <ui class ="nameplateicons">
+                <li class ="icons"> <img src = "image/Cellular Connection.png"></li>
+                <li class ="icons"> <img src = "image/Wifi.png"></li>
+                <li class ="icons"> <img src = "image/Battery.png"></li>
+            </ui>
+        </nav>
```

Comment:
- [ ] Great job building your project to this stage :+1: There are just a few things you need to change. Please remove this entire block of code. It is not important to include that part in your design :100: